File: fsharp-functions/OpenAPI/src/model/JsonSuccessAllOf.fs
namespace OpenAPI.Model

open System
open System.Collections.Generic
open Newtonsoft.Json
open OpenAPI.Model.AnyType

module JsonSuccessAllOf = 

  //#region JsonSuccessAllOf

  [<CLIMutable>]
  type JsonSuccessAllOf = {
    [<JsonProperty(PropertyName = "result")>]
    Result : AnyType;
    [<JsonProperty(PropertyName = "msg")>]
    Msg : AnyType;
  }
  
  //#endregion
  

File: scala-gatling/src/gatling/scala/org/openapitools/client/model/JsonSuccessAllOf.scala

package org.openapitools.client.model


case class JsonSuccessAllOf (
    _result: Option[AnyType],
    _msg: Option[AnyType]
)
object JsonSuccessAllOf {
    def toStringBody(var_result: Object, var_msg: Object) =
        s"""
        | {
        | "result":$var_result,"msg":$var_msg
        | }
        """.stripMargin
}
